fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/MetricMaker.java

```diff
@@ -187,7 +170,7 @@ protected TemplateDruidQuery getMergedQuery(List<String> names) {
      */
     protected PostAggregation getNumericField(String fieldName) {
         // Get the field
-        MetricField field = getDependentQuery(fieldName).getMetricField(fieldName);
+        MetricField field = metrics.get(fieldName).getMetricField();
```

Comment:
This is not the same semantics, is it? In the previous code, it was retrieving an aggregation with a specific name from a metric with that name. Admittedly, the previous code had many assumptions, and I think the new semantics are better, but we should be at least a little careful here, since this is a `protected` method, meaning that others could be relying on it...